resources/recurring-application-charge.js

```diff
@@ -38,4 +38,18 @@ RecurringApplicationCharge.prototype.activate = function activate(id, params) {
     .then(body => body[this.key]);
 };
 
+/**
+ * Customize a recurring application charge (increase the capped amount)
+ *
+ * @param {Number} id Recurring application charge ID
+ * @param {Object} params Recurring application charge properties
```

Comment:
Oooohh, fancy inline code suggestions

```suggestion
 * @param {Number} cappedAmount The new capped amount to apply to the recurring application charge
```